### ISSUE TITLE
stuck in smurf pool pls help!

### ISSUE THREAD

**Issue by Eastofeden1** [ValveSoftware/Dota2-Gameplay#20423]
### Description

Long time ago i lost my main acc, i created new, played 100hours unranked games, than played 1k hours ranked in smurf pool. Then i start playing main pool in ranked games. But when i try to play unranked like turbo or all pick i have problem. I still play with smurfs in unranked. And it takes from 30 min to few hour to find a single turbo game. I cant take it any more. PLs help!

### Example Match ID (and possibly Timestamp)

7869114361

### Screenshots

![Screenshot_5](https://github.com/user-attachments/assets/8043ea1c-4573-417f-9e3d-d8decb113602)
![image](https://github.com/user-attachments/assets/62f27b6c-c6a5-44da-a061-607463d73a8f)
![image](https://github.com/user-attachments/assets/1119d338-2449-4d1e-b8e5-d57bd4449263)
![image](https://github.com/user-attachments/assets/43ce0774-595c-41ae-bcd5-5f3a4a83625d)
![image](https://github.com/user-attachments/assets/d93ba248-bdfd-4407-b10c-800d3444739e)

**Comment by IgorHoruzha** [comment]
Do You still have the issue?

**Comment by Eastofeden1** [comment]
> Do You still have the issue?
Yes i do

**Comment by IgorHoruzha** [comment]
Are there any difference if You play in a party? Could You provide regions You usually play? Do us usually play on a surf account now?

**Comment by Eastofeden1** [comment]
> Are there any difference if You play in a party? Could You provide regions You usually play? Do us usually play on a surf account now?

If i play party with mate who have old account - enemy also will have same guy with old acc. But my team as always only new accs. I`m playing West and south europe, but sometimes i choose russia to find game faster(no reaction usually). No im not playing other accounts

**Comment by IgorHoruzha** [comment]
Have You ever changed the internet provider that You usually use with this account?

**Comment by Eastofeden1** [comment]
> Have You ever changed the Internet provider or a City that You usually use with this account?

no i didnt

**Comment by Eastofeden1** [comment]
> Have You tried to play in a team of 5?

yes, but nothing changes. Still hard to find a game. Mb a bit easier cuz it looking for a 5 man team.

**Comment by IgorHoruzha** [comment]
Do You use Dota+ to check players skill range You usually play with?

**Comment by Eastofeden1** [comment]
> Do You use Dota+ to check players skill range You usually play with?

yes, its alway super difference in skill. I never played with same skill bracket in turbo

**Comment by IgorHoruzha** [comment]
> yes, its alway super difference in skill. I never played with same skill bracket in turbo

Does skipping help? Could You provide an image of typical Dota+ match statistic, please?

**Comment by Eastofeden1** [comment]
> Could You also try CO-OP game versus bots, single draft, random draft, and ability draft, please? Provide the Dota+ data of each type of matches, please.

sure, but it will takes some time

**Comment by IgorHoruzha** [comment]
![image](https://github.com/user-attachments/assets/4f5381a0-d673-4f49-a640-6d1e9c59974f)
![image](https://github.com/user-attachments/assets/3202790b-e3d8-4200-ba93-632f4bc85c49)
![image](https://github.com/user-attachments/assets/ff09be69-5643-4c30-b8e6-83d518cb4fe5)

I also have experience when I play with the account that have an issue that is possible the same in party of 2 and 3, the time is necessary to find a game became almost usual.
In party of two skill range was Acceptable.
So, it may be useful to make some attention on playing with somebody, and consequences.
#17396

**Comment by IgorHoruzha** [comment]
The interesting that Player Behavior Scores were not Similar, I suppose it is a bug.

**Comment by Eastofeden1** [comment]
![1і](https://github.com/user-attachments/assets/14db7fe7-4e22-4529-ba14-158d5502ce78)
 1st attempt turbo game

**Comment by IgorHoruzha** [comment]
Could You try Queue Again, for several times, please?

**Comment by Eastofeden1** [comment]
> Could You try Queue Again, for several times, please?

im cansel game with Dota+ function, and still searching game for 15 min right now

**Comment by IgorHoruzha** [comment]
Could You provide Your rank confidence value, please?

**Comment by Eastofeden1** [comment]
> Could You provide Your rank confidence value, please?

![Screenshot_2](https://github.com/user-attachments/assets/d94d1a43-3fd1-415f-a801-85d442ee62d2)
![Screenshot_1](https://github.com/user-attachments/assets/18400082-8719-482a-a4ef-11cbfc86873c)

**Comment by Eastofeden1** [comment]
> Could You try Queue Again, for several times, please?

![2s](https://github.com/user-attachments/assets/89805b4a-6f88-4b96-b47a-425f724b69b6)
second attempt

**Comment by IgorHoruzha** [comment]
>second attempt

It looks better in general.

**Comment by Eastofeden1** [comment]
![3s](https://github.com/user-attachments/assets/d1174bef-642e-40c2-bca0-e335593deff0)
3rd attempt 
and i cansel searching for now

**Comment by IgorHoruzha** [comment]
The best attempt was the second one, pretty interesting, but any way 15  is a long queue.

**Comment by Eastofeden1** [comment]
> The best attempt was the second one, pretty interesting, but any way 15 is a long queue.

last game i was searching like 30 min

**Comment by Eastofeden1** [comment]
![4s](https://github.com/user-attachments/assets/e9c45aaf-618c-43a1-a6f7-1e5b02adfd8c)
coop game with bots

**Comment by IgorHoruzha** [comment]
What about Single Draft?

**Comment by Eastofeden1** [comment]
> What about Single Draft?

![image](https://github.com/user-attachments/assets/1cde30f2-8a67-400d-bd98-28cfa45cc457)
im tired boss

**Comment by IgorHoruzha** [comment]
What servers are chosen?

**Comment by IgorHoruzha** [comment]
Would You mind If we try to find a game in a party?

**Comment by Eastofeden1** [comment]
> What servers are chosen?

europe west and east

**Comment by Eastofeden1** [comment]
> Would You mind If we try to find a game in a party?

lets try 
1229557033

**Comment by IgorHoruzha** [comment]
>lets try
1229557033

Playing in a party with My main account essentially reduce the time is necessary. The interesting, during the match my ethernet turned off, and also some the teammate and enemies Ethernet turned off. I managed to solve the issue by switching to Phone Ethernet using USB tethering helped me not to leave. So the game was ended with a ping of 90-100ms.
![image](https://github.com/user-attachments/assets/abd65b92-5918-423c-bedc-dcff7854068b)

**Comment by Eastofeden1** [comment]
My internet provider has some issues...

**Comment by IgorHoruzha** [comment]
>My internet provider has some issues...

I switched to Phone Ethernet using USB tethering.

**Comment by IgorHoruzha** [comment]
Could You provide Game Account Client info, please?  Make sure you replace your steam id or custom url.
https://steamcommunity.com/id/*PLACE_YOUR_STEAM_ID_HERE*/gcpd/570/?category=Account&tab=GameAccountClient

**Comment by Eastofeden1** [comment]
> Could You provide Game Account Client info, please? Make sure you replace your steam id or custom url. https://steamcommunity.com/id/*PLACE_YOUR_STEAM_ID_HERE*/gcpd/570/?category=Account&tab=GameAccountClient

![image](https://github.com/user-attachments/assets/d49d494e-8a5e-486b-83ea-c20f2b87dfec)

**Comment by IgorHoruzha** [comment]
Could You provide GACFlagsAudit, FeedEvents and info, please? Make sure you replace your steam id or custom url.
https://steamcommunity.com/id/*PLACE_YOUR_STEAM_ID_HERE*/gcpd/570/?category=Account&tab=GACFlagsAudit
https://steamcommunity.com/id/*PLACE_YOUR_STEAM_ID_HERE*/gcpd/570/?category=Account&tab=FeedEvents

**Comment by Eastofeden1** [comment]
![image](https://github.com/user-attachments/assets/25ee2dce-f755-42ef-accb-bddae1225bf3)
GACFlagsAudit

**Comment by Eastofeden1** [comment]
![image](https://github.com/user-attachments/assets/ccfa07c5-5810-4907-9000-6c40965ca246)
![image](https://github.com/user-attachments/assets/4b05e95c-aa58-4a3e-9a37-f453f1e625e1)
![image](https://github.com/user-attachments/assets/ae74841c-9674-4f30-adfb-186f92b1d658)
FeedEvents
